Documentation/connectors/oauth.md

```diff
@@ -0,0 +1,64 @@
+# Authentication using Generic OAuth 2.0 provider
```

Comment:
We've relocated the documentation to a new repository: https://github.com/dexidp/website